packages/core/src/web/app/components/ImageEditPanel/components/Sider/CornerRadius.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { memo } from 'react';
+
+import { QuestionCircleOutlined } from '@ant-design/icons';
+import { Col, Form, InputNumber, Row, Slider } from 'antd';
+
+import useI18n from '@core/helpers/useI18n';
+
+import { useImageEditPanelStore } from '../../store';
+
+import styles from './PanelContent.module.scss';
+
+const CornerRadius = (): React.JSX.Element => {
+  const { image_edit_panel: lang } = useI18n();
+  const cornerRadius = useImageEditPanelStore((state) => state.cornerRadius);
+  const setCornerRadius = useImageEditPanelStore((state) => state.setCornerRadius);
```

Comment:
only pick `cornerRadius` and `setCornerRadius` to avoid unnecessary re-renders. Same for other files